Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -368,7 +382,7 @@ import BraintreeDataCollector
             hostAndPath.append("/")
         }
         
-        if hostAndPath != BTPayPalRequest.callbackURLHostAndPath {
+        if hostAndPath != BTPayPalRequest.callbackURLHostAndPath && (payPalRequest as? BTPayPalVaultRequest)?.universalLink == nil {
```

Comment:
curious why we add the check for `universalLink` being `nil` here 👀

Reply:
Good question, we only care about (and want to return false here for `isValidURLAction`) if the `hostAndPath` is not the `callbackURLHostAndPath` and if the `universalLink` is nil. That combination would mean that we aren't using the UL flow and the `hostAndPath` is invalid we wouldn't have a valid URL action. If we are using the UL flow we can essentially skip this check since it's not relevant to that flow.

Reply:
This line is a little confusing to read IMO. I think it will clear up in DTBTSDK-3751